src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataTypeAttributeAdapter.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
     /// A validation adapter that is used to map <see cref="DataTypeAttribute"/>'s to a single client side validation
     /// rule.
     /// </summary>
-    public class DataTypeAttributeAdapter : DataAnnotationsModelValidator
+    public class DataTypeAttributeAdapter : DataAnnotationsClientModelValidator
```

Comment:
Good idea, we can do that once we do the moving of this code base to under rendering etc. Let me know if you want to do this right away.